rust-toolchain

```diff
@@ -0,0 +1 @@
+nightly-2019-08-16
```

Comment:
I've pinned the toolchain to `nightly-2019-08-16` because the build fails for some of the rustdoc tests on current nightly with an error message about not being able to find `proc_macro_call_0`. Seems related to this: https://github.com/rust-lang/rust/issues/63651